lib/core/boot/boot.ts

```diff
@@ -79,11 +84,38 @@ export const bootstrap = (): FluxifyServer => {
 						if (targetRoute.schema.body) body = targetRoute.schema.body.parse(body);
 					}
 
+					if (useCache) {
+						if (global.server.cache.length > config.cacheLimit) {
+							global.server.cache.pop();
```

Comment:
Maybe instead of removing the last (latest) cache item, remove the first (oldest)?